src/Adapter/Attribute/Repository/AttributeRepository.php

```diff
@@ -169,7 +155,7 @@ private function getAttributeCombinationAssociations(array $combinationIds): arr
      * @param int[] $attributeIds
      * @param int $langId
      *
-     * @return array<int, array<string, mixed>>
+     * @return array
```

Comment:
use psalm notation to define array elements type (like it previously was)

Reply:
already done